Modules/_io/bufferedio.c

```diff
@@ -428,6 +428,18 @@ buffered_clear(buffered *self)
     return 0;
 }
 
+static PyObject *
+buffered_at_fork_reinit(buffered *self, PyObject *Py_UNUSED(ignored))
+{
+    int ret = _PyThread_at_fork_reinit(&self->lock);
+    self->owner = 0;
+    if (ret == 0) {
+        return Py_True;
```

Comment:
Shouldn't this be `Py_RETURN_TRUE` and `Py_RETURN_FALSE` so that the reference counts get incremented?